infra/app/orders.bicep

```
@@ -0,0 +1,33 @@
+param location string = resourceGroup().location
+param tags object = {}
+
+param containerAppsEnvironmentName string
+param containerRegistryName string
+param imageName string = ''
+param name string = ''
+param serviceName string = 'orders'
+param managedIdentityName string = ''
+
+module orders '../core/host/container-app.bicep' = {
```

```suggestion
module subscriber '../core/host/container-app.bicep' = {
```

Whole file should rename references of `orders` to `subscriber`